packages/now-cli/src/commands/certs/ls.ts

```diff
@@ -130,13 +138,18 @@ function formatCertCn(cn: string, multiple: boolean) {
   return multiple ? `${chalk.gray('-')} ${chalk.bold(cn)}` : chalk.bold(cn);
 }
 
-function formatCertFirstCn(time: Date, cert: Cert, cn: string, multiple: boolean): string[] {
+function formatCertFirstCn(
+  time: Date,
+  cert: Cert,
+  cn: string,
+  multiple: boolean
+): string[] {
   return [
     cert.uid,
     formatCertCn(cn, multiple),
     formatExpirationDate(new Date(cert.expiration)),
     cert.autoRenew ? 'yes' : 'no',
-    chalk.gray(ms(time.getTime() - new Date(cert.created).getTime()))
+    chalk.gray(ms(time.getTime() - new Date(cert.created).getTime())),
```

Comment:
Nice catch with `ms`! Should this also say "ago" like the `rm` command?

Reply:
I guess rm is using Date.now() but this one is given a time variable so maybe not 🤔